tests/test_raster.py

```diff
@@ -1061,14 +1061,32 @@ def test_copy(self, example: str) -> None:
         assert r.raster_equal(r2)
 
         # -- Fifth test: check that the new_array argument works when providing a new dtype ##
+        # For an integer dataset cast to float, or opposite (the exploradores dataset will cast from float to int)
         if "int" in r.dtype:
             new_dtype = "float32"
         else:
             new_dtype = "uint8"
-        r2 = r.copy(new_array=r_arr.astype(dtype=new_dtype))
 
+        # This should work all the types by default due to automatic casting
```

Comment:
typo "all the types" -> "for all the types"

Reply:
fixed